### ISSUE TITLE
Accessibility updates needed for tree.html

### ISSUE THREAD

**Issue by snidersd** [jupyter/notebook#3219]
Also see: [jupyter/notebook#1801](https://github.com/jupyter/notebook/issues/1801)
After running the Dynamic Assessment Plugin for Chrome and the Digital Content Checker on tree.html the following issues were found and can be corrected as indicated:

1. Line 82, if the <title="{% trans %}Refresh notebook list{% endtrans %}"> is actually a label change it to . If it is a tooltip add an <aria-label="Refresh Button">.
2. Line 90, if the <title="{% trans %}Select All / None{% endtrans %}"> is actually a label change it to . If it is a tooltip add an <aria-label="Select All / None Button">.
3. Line 91, add  .
4. Line 106, add alt text to the folder icon
5. Line 111, change
6. to since breadcrumbs are a form of navigation. .
7. Line 133, if the <title="{% trans %}Refresh running list{% endtrans %}"> is actually a label change it to . If it is a tooltip add an <aria-label="Refresh">
8. Line 34-40, either change the <title=""> to <aria-label=""> if it is a label or add the <aria-label=> if it is meant to be a tooltip.

**Comment by takluyver** [comment]
Closed by #3271, I think. Some of the details of the issue seem to have gone missing in Github formatting, so if there are still changes needed, please open a new issue about them.
